services-camel-app/src/test/java/edu/unc/lib/boxc/services/camel/thumbnails/ThumbnailRouterTest.java

```diff
@@ -40,11 +50,26 @@ public void requestSentTest() throws Exception {
         request.setAgent(agent);
         request.setFilePidString(pid.toString());
         var body = ThumbnailRequestSerializationHelper.toJson(request);
-        template.sendBody(body);
+        template.sendBodyAndHeader(body, RESOURCE_TYPE, Cdr.FileObject.getURI());
```

Comment:
I don't think we should be passing the resourceType header in these tests since that wouldn't be the case in the deployed application